pages/api/auth/[...nextauth].js

```diff
@@ -31,9 +34,17 @@ export default NextAuth({
       maxAge: 24 * 60, // 1 hour max life for login request
     }),
   ],
+  // adapter: Adapters.Prisma.Adapter({
```

Comment:
What is this for again?

Reply:
Anish is working on getting the user role added to the session so we can verify user role permissions.  I was researching into using the prisma adapter as an option, but wasn't able to get it to work.